fix(typo): Some Avgi typo fixes (#12573)
endless-sky/endless-sky: data/avgi/avgi weapons.txt

	"mass" 3
	"ophrys capacity" -1
	description "The Ophrys is a minimally-guided nuclear torpedo derived from the Orchid Nuclear Missile. Little more than the Orchid's terminal stage with a layer of ablative armor added to the front, it allows even small drones to pose a credible threat to a capital warship."

> 	description "	Unlike the other nuclear missiles employed by the Avgi, the Ophrys carries only enough propellant to make small adjustments to its trajectory. It instead relies on the speed of the bomber carrying it, trading an independent propulsive capability for enough armor to improve the odds of surviving antimissile fire during its terminal path."

Reviewer: This is supposed to be drawing a distinction between the Ophrys as an unguided bomb and their actual missiles.

Reviewer 2: Hmm... maybe there's another word then that would be better than "missile"? After all, the dictionary definition I'm getting for a missile is "an object or weapon that is fired, thrown, dropped, or otherwise projected at a target; a projectile," which would seem to include an unguided bomb.

Reviewer: I don't think it needs to be that deep

Reviewer 2: Well, when I read that it implies that the Ophrys either (a) isn't nuclear, (b) isn't a missile, (c) isn't employed by the Avgi, or (d) some combination of the three. 

Since it clearly is a nuclear missile employed by the Avgi, it doesn't work properly and just confuses the player without getting the intended point across.

How about this instead?

```suggestion
	description "	Unlike the larger, guided nuclear missiles the Avgi employ, the Ophrys carries only enough propellant to make small adjustments to its trajectory. It instead relies on the speed of the bomber carrying it, trading an independent propulsive capability for enough armor to improve the odds of surviving antimissile fire during its terminal path."
```

Reviewer: I'm not too picky about it. "Other" is probably the simplest fix.

> 	description "	Unlike the larger, guided nuclear missiles the Avgi employ, the Ophrys carries only enough propellant to make small adjustments to its trajectory. It instead relies on the speed of the bomber carrying it, trading an independent propulsive capability for enough armor to improve the odds of surviving antimissile fire during its terminal path."

Reviewer: I think this is still more awkward.

Author: Looking at this again (while not extremely nervous), it dose seem a bit awkward...

Also, has anyone else nervously applied a suggestion or is it just me?

Yes, I really was that nervous (and I still am to some extent).

Edit: while thinking about it some more, I also think the way it was before ("Unlike the other nuclear missiles employed by the Avgi...") is also a bit awkward...

> 	description "	Unlike the larger, guided nuclear missiles the Avgi employ, the Ophrys carries only enough propellant to make small adjustments to its trajectory. It instead relies on the speed of the bomber carrying it, trading an independent propulsive capability for enough armor to improve the odds of surviving antimissile fire during its terminal path."

Reviewer 3: Changing this one back to the way it was. The logs are a genuine error. This just seems like a preferential change.
```suggestion
	description "	Unlike the nuclear missiles employed by the Avgi, the Ophrys carries only enough propellant to make small adjustments to its trajectory. It instead relies on the speed of the bomber carrying it, trading an independent propulsive capability for enough armor to improve the odds of surviving antimissile fire during its terminal path."
```
